stat.go

```diff
@@ -76,32 +76,28 @@ func CDF(q float64, c CumulantKind, x, weights []float64) float64 {
 		return 1
 	}
 
-	var sumWeights float64
-	if weights == nil {
-		sumWeights = float64(len(x))
-	} else {
-		sumWeights = floats.Sum(weights)
-	}
+	sumWeights := floats.Sum(weights)
```

Comment:
This substitution is not correct. This gives sumWeights of zero rather than len(x).